docs/jsonapi-spec.md

```diff
@@ -592,6 +593,87 @@ See [Multi-Document Failure Considerations](#multi-document-failure-consideratio
 
 If an error occurs the command will not return `data`.
 
+
+### findCollection Command
+
+`findCollection` returns all valid collection tables from a given namespace.
+
+*Syntax:*
+
+```bnf
+<findCollection-command> ::= findCollection <findCollection-command-payload> 
+<findCollection-command-response> ::= <paginated-document-response> 
+
+<findCollection-command-payload> ::= <namespace>?
+                                     <projection-clause>? 
+                                     <sort-clause>? 
+                                     <findCollection-command-options>?
+
+<findCollection-command-options> ::= (<findCollection-option-name> <findCollection-option-value>,)*
+```
+
+*Sample:*
+
+```json5
+// Find a collection, return complete documents
+{"findCollection" : {} }
+
+// Select where name == "John" and return selected fields
+{"findCollection" : {
+    "namespace" : {"myNamespace"},
+    "projection" : {"???": 1, "???": 1}
```

Comment:
payload is empty, namespace is given in the URL `/v1/{namespace}`

Reply:
fixed in next commit